python/ray/ml/predictor.py

```diff
@@ -0,0 +1,36 @@
+import abc
+from typing import Union
+
+import numpy as np
+import pandas as pd
+from ray.ml.checkpoint import Checkpoint
```

Comment:
```suggestion
import pandas as pd

from ray.ml.checkpoint import Checkpoint
```

Reply:
also since pandas is imported here, does it need to be a common pip dep? 
(alternatively maybe use if type_checking flag?)

Reply:
Using type checking flag for now